### ISSUE TITLE
AmbiguousBlockAssociation should not apply to Rails scopes

### ISSUE THREAD

**Issue by andreydeineko** [bbatsov/rubocop#4246]
```ruby
class User < ApplicationRecord
  scope :active, -> { where(active: true) }
end
```
Running rubocop I get the following warning:

> Parenthesize the param `-> { where(active: true) }` to make sure that the block will be associated with the `->` method call.

I am sure this check should allow stabby lanbda syntax and not force Rails developers to write ugly things like this one 😉
```
 scope :active, (-> { where(active: true) })
```

**Comment by bbatsov** [comment]
That's a bug which was already fixed on `master`. See https://github.com/bbatsov/rubocop/pull/4237

**Comment by mensfeld** [comment]
@bbatsov any info when it will be released to rubygems?
